citesphere/src/main/webapp/WEB-INF/views/auth/authorities/editAuth.jsp

```diff
@@ -0,0 +1,47 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+<%@ taglib uri = "http://java.sun.com/jsp/jstl/functions" prefix = "fn" %>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+
+<style>
+.popover {
+	min-width: 300px;
+}
+</style>
+<script>
+
+</script>
+
+<ol class="breadcrumb">
+  <li><a href="<c:url value="/" />">Home</a></li>
+  <li><a href="<c:url value="/auth/authority/list" />">Managed Authority Entries</a></li>
+  <li class="active">${entry.name}</li>
+</ol>
+
+<h2>
+	<strong><c:if test="${not empty entry.name}">${entry.name}</c:if></strong>	  
+</h2>
+
+<c:url value="/auth/authority/${authorityId}/edit/save" var="processingUrl" />
+<form:form action="${processingUrl}" modelAttribute="form" method="POST" id="editForm">
+
+<tr <c:if test="${not fn:contains(entry, 'name') }">style="display:none;"</c:if>>
+<td>Name</td>
+<td>
+<form:input path="Name" type="text" class="form-control" placeholder="Name" value="${not empty entry.name ? form.name : entry.name}" /></td>
```

Comment:
if the entry has a name, you show what's in the form? otherwise the name of the entity (which is empty)? That doesn't seem right.